SPECIFICATION.md

```diff
@@ -312,6 +333,7 @@ TODO
 | license | The SPDX identifier for the license of this font. Create multiple `license` entries if the font is published under multiple licenses. | 1-n | `Apache-2.0` |
 | letter_spacing | Global letter space width for every glyph. Same unit as for polylines. If not specified, 0 is assumed (no space). | 0-1 | `1.8` (= 20% of font height) |
 | line_spacing | Vertical baseline distance of multiline texts. Same unit as for polylines. If not specified, 9 is assumed (no space). | 0-1 | `16` (= 178% of font height) |
+| monospace_width | Specifying this header marks the font as a monospace font. The value specifies the width of all glyphs (excluding global spacing). | 0-1 | `6` (= 66% of font height) |
```

Comment:
With proper rounding, it's actually 67% :)

Reply:
Pfff :)